lib/config_cat/user.ex

```diff
@@ -28,7 +28,6 @@ defmodule ConfigCat.User do
   end
 
   defp normalize(attribute) do
-    attribute
-    |> to_string()
+    to_string(attribute)
```

Comment:
#minor I think we could eliminate the `normalize` function entirely and inline the `to_string` call in the one place it's called from, but not a big deal.